alca-app/src/components/ConfirmUnstakeModal.jsx

```diff
@@ -23,8 +23,8 @@ export function ConfirmUnstakeModal({ unstakePosition, onClose, handleUnstake })
     const ModalContent = () => (
         <Box sx={{ my: 2 }}>
             <Typography variant="p">
```

Comment:
My understanding is that by default the Typography component variant always renders a paragraph so there's no need to ask for it.

Reply:
Sorry, I should have included a screenshot, I think this is related to rewards being empty

<img width="1122" alt="Captura de Pantalla 2023-01-26 a la(s) 12 48 36" src="https://user-images.githubusercontent.com/4819518/214882240-26b84fa9-c17f-447c-a5f5-e12fe85699db.png">